lib/bookmark/archives.ex

```diff
@@ -30,6 +30,10 @@ defmodule Bookmark.Archives do
     Repo.all(query)
   end
 
+  def get_archives_by_user(user) do
```

Comment:
We should add test coverage for this function

Reply:
Done!